[2024-01-17T19:45:03.987-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:43:50.589390+00:00 [queued]>
[2024-01-17T19:45:03.990-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:43:50.589390+00:00 [queued]>
[2024-01-17T19:45:03.990-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T19:45:04.000-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-17 22:43:50.589390+00:00
[2024-01-17T19:45:04.004-0300] {standard_task_runner.py:57} INFO - Started process 89337 to run task
[2024-01-17T19:45:04.006-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'df', 'scheduled__2024-01-17T22:43:50.589390+00:00', '--job-id', '2597', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpkwe6x1fk']
[2024-01-17T19:45:04.006-0300] {standard_task_runner.py:85} INFO - Job 2597: Subtask df
[2024-01-17T19:45:04.024-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:43:50.589390+00:00 [running]> on host desktop
[2024-01-17T19:45:04.051-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T22:43:50.589390+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T22:43:50.589390+00:00'
[2024-01-17T19:45:04.587-0300] {logging_mixin.py:154} INFO - Called | 1705531504.584849 | df
[2024-01-17T19:45:04.594-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-17T19:45:04.595-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T19:45:04.598-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=df, execution_date=20240117T224350, start_date=20240117T224503, end_date=20240117T224504
[2024-01-17T19:45:04.618-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T19:45:04.626-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
